content/en/agent/configuration/secrets-management.md

```diff
@@ -650,6 +650,20 @@ To refresh manually, use:
 datadog-agent secret refresh
 ```
 
+### Automatic secrets refresh on API key failure / invalidation
+
+Starting in Agent version v7.74, the Agent can automatically refresh secrets when it detects an invalid API key. This happens when the Agent receives a 403 Forbidden response from Datadog or when the periodic health check detects an invalid API key.
+
+To enable this feature, set `secret_refresh_on_api_key_failure_interval` (in minutes) in your `datadog.yaml` file:
```

Comment:
```suggestion
To enable this feature, set `secret_refresh_on_api_key_failure_interval` to an interval in minutes in your `datadog.yaml` file. Set to `0` to disable (default).

This interval is the minimum amount of time between 2 refreshes to avoid spamming your secrets management solution when an invalid API key is detected.
```